core/systems/graphite/mix/gen_submissions.ex

```diff
@@ -0,0 +1,80 @@
+defmodule Mix.Tasks.Eyra.Graphite.Gen.Submissions do
+  @moduledoc """
+  Generate a number of submissions for a `leaderboard`.
+
+  Usage:
+    $ mix eyra.graphite.gen.submissions -l LEADERBOARD -n AMOUNT -p PREFIX
+  """
+
+  use Mix.Task
+
+  alias Systems.Graphite
+
+  @options [
+    strict: [leaderboard: :integer, amount: :integer, prefix: :string],
+    aliases: [l: :leaderboard, n: :amount, p: :prefix]
+  ]
+
+  @shortdoc "Generate Graphite submissions"
+  def run(args) do
+    Mix.Task.run("app.start")
+
+    case OptionParser.parse(args, @options) do
+      {[leaderboard: id, amount: amount, prefix: prefix], _, _} ->
+        create_submissions(id, amount, prefix)
+
+      {[leaderboard: id, amount: amount], _, _} ->
+        create_submissions(id, amount, "auto-gen")
+
+      _ ->
+        print_missing_arguments()
+    end
+  end
+
+  defp create_submissions(leaderboard_id, amount, prefix) do
+    leaderboard = Graphite.Public.get_leaderboard!(leaderboard_id, [{:tool, :auth_node}])
+
+    multi = Ecto.Multi.new()
+
+    {:ok, result} =
+      Enum.reduce(1..amount, multi, fn i, multi ->
+        submission =
+          Core.Factories.build(:graphite_submission, %{
+            tool: leaderboard.tool,
+            auth_node: leaderboard.tool.auth_node,
```

Comment:
Consider to create a user for every submission and add the user as :owner to the auth_node. Otherwise the leaderboard will have the same user name for every submissions